ts/Dashboards/EditMode/Sidebar.ts

```diff
@@ -27,6 +27,8 @@ import EditRenderer from './EditRenderer.js';
 import Bindings from '../Actions/Bindings.js';
 import GUIElement from '../Layout/GUIElement.js';
 import Layout from '../Layout/Layout.js';
+import type Component from '../Component/Component.js';
```

Comment:
Can be removed, also could you please separate the `type` imports from the other imports?

Reply:
It's surprising, because the linter skipped it :D 

Thanks! I will fix it!